### PR TITLE
Feature/back 9 email verification 

### PR DESCRIPTION
**Pull request by IWorkForFood**

### CODE REVIEW

File: src/apps/authentication/admin.py [hunk 1]
```diff
@@ -1,2 +1,9 @@
-
 # Register your models here.
+
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+from .models import User
+
+admin.site.register(User, UserAdmin)
```
Comment (egorgur) [inline src/apps/authentication/admin.py]: For some reason the User.is_verified is not manageable through django admin page

File: src/config/settings/rest_framework.py [hunk 2]
```diff
@@ -47,4 +47,13 @@
...
+}
+
+
+#Settings for connection to the gmail account
+EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
+EMAIL_USE_TLS = True
+EMAIL_HOST = 'smtp.gmail.com'
+EMAIL_PORT = 587
+EMAIL_HOST_USER = 'anynoteconfirm@gmail.com'
+EMAIL_HOST_PASSWORD = 'mcqb jsgo zvki aznt'
```
Comment (egorgur) [inline src/config/settings/rest_framework.py]: hardcoded password not in local settings
I would suggest to put it in local settings files and set the exemplar one in template settings also

File: src/apps/authentication/api/utils.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    """
+
+    def __init__(self, email):
+        self.email = email
+        threading.Thread.__init__(self)
+
+    def run(self):
+        self.email.send()
+
+class Util:
```
Comment (egorgur) [inline src/apps/authentication/api/utils.py]: To abstract and global name for a class.
This class only manages the mail-theme functionality for now and it should stay like this.
Do not mix differently themed functionality into one class see SOLID principles for this. 
Considering the Above ^, rename this `Util` class into somewhat connected to mailing

**Review by egorgur — CHANGES_REQUESTED** [review]

File: src/apps/authentication/admin.py (hunk 1 above)
Comment (IWorkForFood) [reply src/apps/authentication/admin.py]: It is done

File: src/config/settings/rest_framework.py (hunk 2 above)
Comment (IWorkForFood) [reply src/config/settings/rest_framework.py]: It is done

File: src/apps/authentication/api/utils.py (hunk 3 above)
Comment (IWorkForFood) [reply src/apps/authentication/api/utils.py]: It is done

**Review by egorgur — APPROVED** [review]
Great job bro!

**Pull request merged**